global.json

```diff
@@ -1,7 +1,7 @@
-﻿{
+{
   "sdk": {
-    "version": "6.0.0",
+    "version": "8.0.0",
     "rollForward": "latestMajor",
-    "allowPrerelease": false
+    "allowPrerelease": true
```

Comment:
When using `rollForward:latestMajor` it doesn't really make sense to allow prereleases, isn't it?

Reply:
Correct, this is a left over from testing the .net 8.0 pre-release

Will be adjusted in:
PR: #8 
Commit: https://github.com/Jan5366x/MagicFileEncoding/pull/8/commits/30c173b39edbd24219932c7644f119d001253a55